extensions/2.0/Khronos/KHR_materials_anisotropy/README.md

```diff
@@ -0,0 +1,140 @@
+# KHR\_materials\_anisotropy
+
+## Contributors
+
+* Jim Eckerlein, Norbert Nopper, UX3D
+
+## Status
+
+Suggestion
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension defines the anisotropic property of a material as observable with brushed metals for instance.
+An asymetric specular lobe model is introduced to allow for such phenomena.
+The visually distinct feature of that lobe is the elongated appearance of the specular reflection.
+For a single punctual light source, the specular reflection will eventually degenerate into a zero width line in the limit,
+that is where the material is fully anisotropic, as opposed to be fully isotropic in which case the specular reflecation is radially symmetric.
+
+## Extending Materials
+
+Sample values:
+
+```json
+{
+    "materials": [
+        {
+            "extensions": {
+                "KHR_materials_anisotropy": {
+                    "anisotropy": 0.6,
+                    "anisotropyDirection": [0.0, 1.0, 0.0]
+                }
+            }
+        }
+    ]
+}
+```
+
+|                               | Type                                                                 | Description                       | Required                       |
+|-------------------------------|----------------------------------------------------------------------|-----------------------------------|--------------------------------|
+|**anisotropy**                 | `number`                                                             | The anisotropy.                   | No, default: `0.0`             |
+|**anisotropyTexture**          | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo)  | The anisotropy texture.           | No                             |
+|**anisotropyDirection**        | `array`                                                              | The anisotropy direction.         | No, default: `[1.0, 0.0, 0.0]` |
+|**anisotropyDirectionTexture** | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo)  | The anisotropy direction texture. | No                             |
+
+## Anisotropy
+
+Two new material properties are introduced: an explicit anisotropy parameter and the direction in which the specular reflection elongates relative to the surface tangents.
+The anisotropy parameter is a dimensionaless number in `[-1, 1]` and forms an injective relation to the roughness distribution along two orthogonal directions, one of which is the direction parameter and the other the result of crossing the direction and the geometric normal.
```

Comment:
What space is the direction vector in? Same as the normal map, presumably? And what is the distribution function? Our normal roughness is defined by infinitesimal oblate ellipsoids (Trowbridge-Reitz), so presumably this would map to some more complex ellipsoids? And likewise, how does it interact with the normal roughness parameter?